src/shared/copy.h

```diff
@@ -23,6 +23,7 @@ typedef enum CopyFlags {
         COPY_FSYNC       = 1 << 10, /* fsync() after we are done */
         COPY_FSYNC_FULL  = 1 << 11, /* fsync_full() after we are done */
         COPY_SYNCFS      = 1 << 12, /* syncfs() the *top-level* dir after we are done */
+        COPY_ALL_XATTRS  = 1 << 13, /* Preserve all xattrs when copying, not just those in the user namespace */
```

Comment:
maybe add comment that makes explicitly clear that this includes selinux labels (and other security labels)